BUILD.bazel

```diff
@@ -145,6 +145,7 @@ filegroup(
         "frontends/p4-14/ir-v1.def",
         "backends/bmv2/bmv2.def",
         "backends/dpdk/dpdk.def",
+        "//backends/p4tools:p4tools.def",
```

Comment:
Nit: use a relative path, for consistency?

Reply:
Since  'backends/p4tools' is a subpackage (while the other files are not), this should be like this